engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/MultiFormDeploymentDto.ftl

```diff
@@ -33,6 +33,13 @@
         type = "string"
         desc = "The name for the deployment to be created."/>
 
+    <@lib.property
+        name = "deployment-activation-time"
+        type = "string"
+        format = "date-time"
+        desc = "Sets the date on which the process definitions contained in this deployment will be activated. This means that all process
```

Comment:
If we include a date format reference with https://github.com/camunda/camunda-docs-manual/pull/1006, add it here as well. See `TaskQueryDto.ftl` for reference.